<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Meniu</class>
 <widget class="QWidget" name="Meniu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_Menu">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_Meniu">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>MENIU PRINCIPAL</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_pic">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_Events">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Categorii Evenimente</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_Istoric">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Istoric Cont</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_Notificari">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Notificari Evenimente</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton_LogOut">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>LogOut</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_Categorii">
      <widget class="QLabel" name="label_Categorii">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>CATEGORII EVENIMENTE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_BackEvents">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>110</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Sport">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Sport</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Muzica">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>210</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Muzica</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Teatru">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>270</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Teatru</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_Evenimente">
      <widget class="QLabel" name="label_Evenimente">
       <property name="geometry">
        <rect>
         <x>212</x>
         <y>40</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>EVENIMENTE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_BackEvenimente">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>110</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_Evenimente">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>400</width>
         <height>150</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Detalii">
       <property name="geometry">
        <rect>
         <x>452</x>
         <y>100</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Detalii</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_Detalii">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>309</y>
         <width>400</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Favorit">
       <property name="geometry">
        <rect>
         <x>452</x>
         <y>140</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Favorite</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_CumparaBilet">
       <property name="geometry">
        <rect>
         <x>452</x>
         <y>220</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>CumparaBilet</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Defavorite">
       <property name="geometry">
        <rect>
         <x>452</x>
         <y>180</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>EliminaFavorit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>270</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Detalii Eveniment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_Istoric">
      <widget class="QLabel" name="label_Istoric">
       <property name="geometry">
        <rect>
         <x>232</x>
         <y>39</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ISTORIC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_BackIstoric">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>110</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_EvenimenteFavorite">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>EvenimenteFavorite</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_EvenimenteFavorite">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>130</y>
         <width>300</width>
         <height>130</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_BileteCumparate">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>BileteCumparate</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_BileteCumparate">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>270</y>
         <width>300</width>
         <height>130</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_Notificari">
      <widget class="QLabel" name="label_Notificari">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>NOTIFICARI</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_BackNotificari">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>110</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Noutati">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>110</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Noutati</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Statistici">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>110</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>Statistici</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_Noutati">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>400</width>
         <height>130</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_Statistici">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>270</y>
         <width>400</width>
         <height>130</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
